filecoin-proofs/examples/zigzag.rs

```diff
@@ -501,10 +514,37 @@ fn main() {
     let taper_layers = value_t!(matches, "taper-layers", usize).unwrap_or(layers);
     let use_tmp = !matches.is_present("no-tmp");
     let dump_proofs = matches.is_present("dump");
-    let groth = matches.is_present("groth");
-    let bench = !matches.is_present("no-bench");
+    let no_bench = matches.is_present("no-bench");
+    let bench_only = matches.is_present("bench-only");
+    let bench = if no_bench {
+        if bench_only {
+            panic!("Cannot specify both --bench-only and --no-bench.");
```

Comment:
there is a much nicer way of specifying this: https://docs.rs/clap/2.32.0/clap/struct.Arg.html#method.conflicts_with

Reply:
I figured as much, thanks for the reference.